src/pages/javascript/switch-statements/index.md

```diff
@@ -2,14 +2,71 @@
 title: Switch Statements
 ---
 ## Switch Statements
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A `switch` statement in programming is similar to an `if-else` statement, but has the benefit of sometimes being easier to read when there are a lot of conditions, and it allows a `default` block, that will be executed if none of the other conditions are true.
```

Comment:
Let's break this sentence into two to keep it simple: "... a lot of conditions. Also, it allows for a `default` block, ..."

Reply:
Fixed, thanks!